forge-gui/res/cardsfolder/upcoming/festerleech.txt

```diff
@@ -0,0 +1,9 @@
+Name:Festerleech
+ManaCost:B
+Types:Creature Zombie Leech
+PT:1/1
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player | CombatDamage$ True | Execute$ TrigMill | TriggerZones$ Battlefield | TriggerDescription$ Whenever CARDNAME deals combat damage to a player, you mill two cards.
+A:AB$ Pump | Cost$ 1 B | NumAtt$ +2 | NumDef$ +2 | ActivationLimit$ 1 | SpellDescription$ Festerleech gets +2/+2 until end of turn. 
+SVar:TrigMill:DB$ Mill | NumCards$ 2 | Defined$ You

```

Comment:
I would recommend grouping this above the ability so its right next to the trigger that cares about it.